modules/apps/change-tracking/change-tracking-rest-test/src/testIntegration/java/com/liferay/change/tracking/rest/resource/v1_0/test/CTCollectionResourceTest.java

```diff
@@ -274,6 +274,32 @@ protected CTCollection testGetCTCollection_addCTCollection()
 		return ctCollectionResource.postCTCollection(randomCTCollection());
 	}
 
+	@Override
+	protected CTCollection testGetCtCollectionsByClassPage_addCTCollection(
+			CTCollection ctCollection)
+		throws Exception {
+
+		CTCollection postCTCollection = ctCollectionResource.postCTCollection(
+			ctCollection);
+
+		com.liferay.change.tracking.model.CTCollection
+			serviceBuilderCTCollection =
+				_ctCollectionLocalService.getCTCollection(
+					postCTCollection.getId());
+
+		serviceBuilderCTCollection.setCreateDate(ctCollection.getDateCreated());
+		serviceBuilderCTCollection.setModifiedDate(
+			ctCollection.getDateModified());
+		serviceBuilderCTCollection.setShareable(true);
```

Comment:
Do we need to make it shareable? 

Looks like this has the same logic as in `testGetCTCollectionsPage_addCTCollection`. We can extract it to a helper to reduce duplicate code.